pkgs/development/libraries/libpinyin/default.nix

```diff
@@ -1,19 +1,34 @@
-{ stdenv, fetchurl, glib, db, pkgconfig }:
+{ stdenv, fetchurl, fetchFromGitHub, autoreconfHook, glib, db, pkgconfig }:
 
-stdenv.mkDerivation {
-  name = "libpinyin-1.3.0";
+let
+  modelData = fetchurl {
+    url    = "http://downloads.sourceforge.net/libpinyin/models/model12.text.tar.gz";
```

Comment:
It was not, I just copy pasted that link from upstream makefile without noticing.
Thanks for noticing, amended the commit to use the mirror.